src/components/target-pane/target-pane.jsx

```diff
@@ -113,4 +132,8 @@ TargetPane.propTypes = {
     vm: React.PropTypes.instanceOf(VM)
 };
 
+TargetPane.defaultProps = {
+    mediaLibrary: new MediaLibrary()
```

Comment:
Although I've never confirmed this assumption, I am pretty sure this survives rerendering of the component, since the VM is instantiated in the same way and doesn't get recreated on each render. I haven't confirmed this but it should act as a static, since it's only instantiated once for the class. If you don't override it, it should be reusing the same instance for each component.